.github/workflows/docs.yml

```diff
@@ -24,119 +24,11 @@ jobs:
-        uses: octokit/request-action@v2.x
-        with:
-          route: POST /repos/:repository/deployments/:deployment/statuses
-          repository: ${{ github.repository }}
-          deployment: ${{ fromJson(steps.create_deployment.outputs.data).id }}
-          environment: ${{ env.DEPLOYMENT_ENVIRONMENT }}
-          environment_url: ${{ env.DEPLOYMENT_ENVIRONMENT_URL }}
-          log_url: ${{ env.DEPLOYMENT_LOG_URL }}
-          state: in_progress
-          mediaType: '{"previews": ["flash", "ant-man"]}'
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      # Default shell is bash: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell
-      # https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#cli-tools
-      # https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html
-      # https://docs.aws.amazon.com/cli/latest/userguide/cli-services-s3-commands.html#using-s3-commands-managing-objects-sync
-      # https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/sync.html
-      - name: Upload to Amazon S3
-        env:
-          AWS_ACCESS_KEY_ID: ${{ secrets.SDK_S3_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.SDK_S3_ACCESS_KEY }}
-          AWS_DEFAULT_REGION: eu-west-2
-        run: |
-          aws s3 sync --acl public-read doc/api "s3://sdk.ably.com/builds/$GITHUB_REPOSITORY/$DEPLOYMENT_REMOTE_PATH"
-
-      - name: Set Deployment Status to Success
-        uses: octokit/request-action@v2.x
-        with:
-          route: POST /repos/:repository/deployments/:deployment/statuses
-          repository: ${{ github.repository }}
-          deployment: ${{ fromJson(steps.create_deployment.outputs.data).id }}
-          environment: ${{ env.DEPLOYMENT_ENVIRONMENT }}
-          environment_url: ${{ env.DEPLOYMENT_ENVIRONMENT_URL }}
-          log_url: ${{ env.DEPLOYMENT_LOG_URL }}
-          state: success
-          mediaType: '{"previews": ["ant-man"]}'
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      # If the previous step (set deployment status to success) FAILS then this step will ALSO get run.
-      - name: Set Deployment Status to Failure
-        if: ${{ failure() }}
-        uses: octokit/request-action@v2.x
+      - uses: ably/sdk-upload-action@v1
         with:
-          route: POST /repos/:repository/deployments/:deployment/statuses
-          repository: ${{ github.repository }}
-          deployment: ${{ fromJson(steps.create_deployment.outputs.data).id }}
-          environment: ${{ env.DEPLOYMENT_ENVIRONMENT }}
-          environment_url: ${{ env.DEPLOYMENT_ENVIRONMENT_URL }}
-          log_url: ${{ env.DEPLOYMENT_LOG_URL }}
-          state: failure
-          mediaType: '{"previews": ["ant-man"]}'
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            s3AccessKeyId: ${{ secrets.SDK_S3_ACCESS_KEY_ID }}
```

Comment:
Indentation level for this lines is 2 columns too far to the right as this file is conformed to an indentation of 2 spaces.

Reply:
https://github.com/ably/ably-flutter/pull/117/commits/3badd781dd5c2432d19442feb026ee8d132dabd1